BrowserKit/Sources/Common/Constants/StandardImageIdentifiers.swift

```diff
@@ -10,6 +10,7 @@ import Foundation
 public struct StandardImageIdentifiers {
     // Icon size 16x16
     public struct Small {
+        public static let externalLink = "externalLinkSmall"
```

Comment:
I notice that this doesn't match what we have the figma file [here](https://www.figma.com/design/Kv7wWr8vH6AQrGhu9vmjdE/Mobile-Icons?node-id=0-1&t=41X6bB7npySvuD87-0). It does match the github repo, which I assume should be the [source of truth](https://github.com/FirefoxUX/acorn-icons/tree/main/icons/mobile/16). 

@cwzilla should we also update the Figma File or should we point to the github repo going forward?

Reply:
@cwzilla sorry I wasn't clear. It looks like its in the small category, but in the figma file I see it under the large category instead. Should it be moved to 16x16? Also let me know if I'm misreading the file!

Reply:
@cyndichin oh gotcha. just did some digging and it looks like we have 2 variations of the external link icon, so what's in code/repo is correct, the 16x16 icon is just missing in Figma. I'll update it :)

Reply:
thanks!